src/components/ClaimReview/form/DynamicForm.tsx

```diff
@@ -328,19 +252,13 @@ const DynamicForm = ({ sentence_hash, personality, claim, sitekey }) => {
                                     ? "button"
                                     : "submit"
                             }
-                            onClick={
-                                event === ReviewTaskEvents.goback
-                                    ? onClickGoBack
-                                    : event === ReviewTaskEvents.draft
-                                    ? onClickSaveDraft
-                                    : () => {
-                                          //@ts-ignore
-                                          umami?.trackEvent(
-                                              `${event}_BUTTON`,
-                                              "Fact-checking workflow"
-                                          );
-                                      }
-                            }
+                            onClick={() => {
+                                if (
+                                    event === ReviewTaskEvents.goback ||
+                                    event === ReviewTaskEvents.draft
+                                )
+                                    handleSendEvent(event);
```

Comment:
That way we don't get the track of events other than goback and draft

Reply:
Actually we do get the tracking because the submit event calls the handleSendEvent after the validations